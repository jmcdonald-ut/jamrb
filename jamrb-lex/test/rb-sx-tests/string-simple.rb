"Simplest string"
'Also simple string'
