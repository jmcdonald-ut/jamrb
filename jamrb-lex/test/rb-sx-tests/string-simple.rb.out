((Token ((1 0) on_tstring_beg "\""))
 (Token ((1 1) on_tstring_content "Simplest string"))
 (Token ((1 16) on_tstring_end "\""))
 (Token ((1 17) on_nl "\n"))
 (Token ((2 0) on_tstring_beg "'"))
 (Token ((2 1) on_tstring_content "Also simple string"))
 (Token ((2 19) on_tstring_end "'"))
 (Token ((2 20) on_nl "\n")))
